pages/docs/reference/delegated-properties.md

```diff
@@ -153,7 +153,59 @@ fun main() {
 If you want to intercept assignments and "veto" them, use [`vetoable()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.properties/-delegates/vetoable.html) instead of `observable()`.
 The handler passed to the `vetoable` is called _before_ the assignment of a new property value has been performed.
 
-## Storing Properties in a Map
+## Delegating to another property
+
+Since Kotlin 1.4, a property can delegate its getter and setter to another property. Such delegation is available for
+both top-level and class properties (member and extension). The delegate property can be:
+- a top-level property
+- a member or an extension property of the same class
+- a member or an extension property of another class
+
+To delegate a property to another property, use the proper `::` qualifier in the delegate name, for example, `this::delegate` or
+`MyClass::delegate`.
+
+<div class="sample" markdown="1" theme="idea" data-highlight-only>
+
+```kotlin
+
+var topLevelDelegate: Int
+class MyClass(var memberDelegate: Int) {
+    var delegatedToMember: Int by this::memberDelegate
+    var delegatedToTopLevel: Int by ::topLevelDelegate
+    val delegegateToAnotherClass: Int by ClassWithDelegate::anotherClassDelegate
```

Comment:
typo
```suggestion
    val delegateToAnotherClass: Int by ClassWithDelegate::anotherClassDelegate
```

Reply:
Also this likely won't work because the receiver of an unbound property reference must be able to take the receiver of the delegated property, and `ClassWithDelegate` is not a supertype of `MyClass`.

Possible options:
- make `ClassWithDelegate` a supertype of `MyClass`. But then its properties will be inherited, and delegation with an unbound reference won't be required.
- use a bound reference: pass an instance of ClassWithDelegate somehow and delegate to `thatInstance::property`